composer.json

```diff
@@ -38,17 +38,6 @@
     {
       "type": "vcs",
       "url": "https://github.com/christophermancini/php-protobuf"
-    },
-    {
-      "type": "package",
-      "package": {
-        "name": "basho/riak_pb",
-        "version": "2.1.3.0",
-        "dist": {
-          "url": "https://github.com/basho/riak_pb/archive/2.1.3.0.zip",
-          "type": "zip"
-        }
-      }
```

Comment:
It's how all the other clients pull in `riak_pb`. In addition, even a contributor to this project probably doesn't need / want to pull it in, either.